crates/cli-support/src/js/mod.rs

```diff
@@ -2179,6 +2184,10 @@ impl<'a> Context<'a> {
         pairs.sort_by_key(|(k, _)| *k);
         check_duplicated_getter_and_setter_names(&pairs)?;
 
+        if self.should_generate_stack_pointer_shim {
+            self.inject_stack_pointer_shim()?;
```

Comment:
It's fine if this happens during `require_stack_pointer_shim`, and then the context's boolean prevents us from adding it twice.